python/src/fds/protobuf/stach/extensions/v2/StachUtilities.py

```diff
@@ -47,3 +47,33 @@ def get_value(val):
         else:
             return val
 
+    @staticmethod
+    def decompress_column(column):
```

Comment:
Can you please move these two new methods to the new `ColumnStachUtilities.py` as these methods are specific to column-organized structures and not for row-organized STACH?